### PR TITLE
Public lab dependency chart

### PR DESCRIPTION
**Pull request by YogeshSharma01**
Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [ ] code is in uniquely-named feature branch and has no merge conflicts
* [ ] PR is descriptively titled
* [ ] PR body includes `fixes #0000`-style reference to original issue #
* [ ] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

We have a loose schedule of reviewing and pulling in changes every Tuesday and Friday, and publishing changes on Fridays. 

Thanks!

### CODE REVIEW

**Review by jywarren — APPROVED** [review]
Looks awesome!!!! Thank you so much!!!

File: index.html [hunk 1]
```diff
@@ -320,6 +320,12 @@ <h2 class="contributors-head" id="contributer">Contributors</h2>
         <div class="well contributors"><div id="spinner-icon" style="text-align: center;"><i class="fa fa-spinner fa-spin"></i></div><p class="usernames"></p><hr /><p class="avatars text-center"></p></div>
       </div>
 
+      <!-- Dependency chart  -->
+      <div id="dependency-div">
+      <img  class="dependency-chart" src="./images/chart.png" alt="dependency-chart">
```
Comment (jywarren) [inline index.html:325]: Wait, is the chart already in our repository, or do we need to add that file too? Thank you!!!

Comment (YogeshSharma01) [reply index.html:325]: Yes, @jywarren I have added that file too.
Warm Regards,
Yogesh Sharma
Thanks

**Comment by HarshKhandeparkar** [comment]
Can you provide a screenshot or a GIF of the browser window resizing or dev tools responsiveness test?

**Comment by YogeshSharma01** [comment]
Yeah sure @HarshKhandeparkar 

https://user-images.githubusercontent.com/51679200/106933897-78aaeb80-673f-11eb-8dac-08c4dde310e6.mov

**Review by HarshKhandeparkar — APPROVED** [review]

**Comment by daemon1024** [comment]
I think `width:100%;` would streamline all the contents across site since it looks a bit awkward. It is a bit subjective to let me know what y'all think.

**Comment by daemon1024** [comment]
Also I believe if `.png` files are pretty heavy on sites, If we would have a higher quality png, we may convert it to `webp` and this would not impact the website performance negatively.

**Comment by YogeshSharma01** [comment]
Hi, @daemon1024 apologize for my late reply actually I was busy with my exams. Yes agreed with your point width of the picture should be 100% and I also convert the picture from .png to .webp.
Thanks

**Comment by daemon1024** [comment]
No problem @YogeshSharma01. I believe you will need convert the image file first. Just changing the extension won't help 😅. Also can you let me know the source of the file.

**Comment by YogeshSharma01** [comment]
Hi, @daemon1024  sorry for the late reply actually I was busy with my exams and now I have converted the image file.
review it. 
Warm Regards
Yogesh Sharma
Thanks

**Review by daemon1024 — CHANGES_REQUESTED** [review]

File: index.html [hunk 2]
```diff
@@ -320,6 +320,12 @@ <h2 class="contributors-head" id="contributer">Contributors</h2>
         <div class="well contributors"><div id="spinner-icon" style="text-align: center;"><i class="fa fa-spinner fa-spin"></i></div><p class="usernames"></p><hr /><p class="avatars text-center"></p></div>
       </div>
 
+      <!-- Dependency chart  -->
+      <div id="dependency-div">
+      <img  class="dependency-chart" src="./images/ezgif.com-gif-maker.webp" alt="dependency-chart">
```
[the remark this answers is not in the document]
Comment (YogeshSharma01) [reply index.html]: Done

[the remark this answers is not in the document]
Comment (YogeshSharma01) [reply index.html]: Thanks, @daemon1024

**Review by daemon1024 — CHANGES_REQUESTED** [review]
Please rename the webp file to dependency chart and remove png file from the commit. Thanks You.

**Review by daemon1024 — APPROVED** [review]

**Review by jywarren — APPROVED** [review]
Looks nice!!! Thank you!!!

**Comment by jywarren** [comment]
Great work!!!!

**Pull request merged**